src/common/playlist/playlist-manager.js

```diff
@@ -95,8 +95,8 @@ class PlaylistManager {
   playNext(): void {
     PlaylistManager._logger.debug('playNext');
     const next = this._playlist.getNext(true);
```

Comment:
It's not about the change but maybe you will know why doesn't it use the getter next and force to get the next one even the loop config is false?

Reply:
loop it's for the UI and for automatically next playing only. for the api loop it's always true

Reply:
loop it's for the UI and for automatically next playing only. for the api loop it's always true